src/Components/Bill/Bill.css

```diff
@@ -65,12 +54,18 @@
   border: none;
   border-radius: 7px;
   margin: 0.25em;
-  color: hsl(184, 14%, 56%);
+  color: hsl(183, 100%, 15%);
   padding: 0;
   font-family: "Space Mono", monospace;
   font-size: 1.2em;
   font-weight: bold;
   text-align: center;
+  outline: none; 
+}
+
+.custom-tip:focus,
+.input-field:focus {
+  box-shadow: 0 0 0 2px hsl(172, 67%, 45%);
 }
 
 .custom-tip::placeholder {
```

Comment:
Interesting. Can you talk me through this pseudo class looking line when we next gather?